webapp/src/pages/Profile/components/match-history.js

```diff
@@ -20,11 +20,15 @@ export const MatchHistory = (props) => {
     const [values, setValues] = useState([]);
 
     const getHistory = async () => {
-        const res = await Helpers.History.get_match();
-        setValues(res.data.result.reverse());
+        try{
+            const res = await Helpers.History.get_match();
+            setValues(res.data.result.reverse());
+        } catch (e) {
+            console.log(e);
+        }
     };
 
-    useEffect(() => {getHistory();});
+    useEffect(() => {getHistory();}, []);
```

Comment:
@automaths ce n'est pas très lisible, je te propose de suivre la convention

```suggestion
    useEffect(() => {
        getHistory();
    }, []);
```